nuxeo-core/nuxeo-core-storage-mem/src/main/java/org/nuxeo/ecm/core/storage/mem/MemRepository.java

```diff
@@ -266,12 +282,21 @@ public boolean hasChild(String parentId, String name, Set<String> ignored) {
 
     @Override
     public Stream<State> getDescendants(String rootId, Set<String> keys) {
+        return getDescendants(rootId, keys, 0);
+    }
+
+    @Override
+    public Stream<State> getDescendants(String rootId, Set<String> keys, int limit) {
         if (log.isTraceEnabled()) {
             log.trace("Mem: QUERY " + KEY_ANCESTOR_IDS + " = " + rootId);
         }
-        return states.values() //
-                     .stream()
-                     .filter(state -> hasAncestor(state, rootId));
+        Stream<State> stream = states.values() //
```

Comment:
extra ` // ` I guess

Reply:
It's here as a common hack to force some IDEs (Eclipse) to indent this correctly (aligning each call in a fluent style on the same column)